src/python/pants/option/optionable_test.py

```diff
@@ -46,6 +46,7 @@ def check_true(s: str) -> None:
         def check_false(s: str) -> None:
             self.assertFalse(Optionable.is_valid_scope_name_component(s))
 
+        check_true("")
```

Comment:
Ditto on `GLOBAL_SCOPE`

Reply:
Disagreed, for the same reason. This is just about "these are allowed strings", not "these are actual real scopes".